### PR TITLE
Properly configure the project dependencies for TerminalAzBridge

### PR DESCRIPTION
**Pull request by zadjii-msft**
I don't think this is the resolution for #14581, but this can't hurt

### CODE REVIEW

**Comment by zadjii-msft** [comment]
1 / 1 builds succeeded.

/azp run

File: src/cascadia/WpfTerminalControl/TerminalControl.xaml.cs [hunk 1]
```diff
@@ -33,20 +35,6 @@ public TerminalControl()
             this.GotFocus += this.TerminalControl_GotFocus;
         }
 
-        /// <inheritdoc/>
-        protected override AutomationPeer OnCreateAutomationPeer()
```
Comment (DHowett) [inline src/cascadia/WpfTerminalControl/TerminalControl.xaml.cs:37]: ey these seem unrelated -- can they be a separate PR? are thye necessary?

**Comment by zadjii-msft** [comment]
3/3 builds passed it a row? Man I don't wat to get ahead of myself but I think this was it

**Review by DHowett — APPROVED** [review]

**Review by lhecker — APPROVED** [review]
